docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,35 +1,372 @@
+   ```
+
+When finished, [continue to the next step](#step-4-set-up-environment-variables-and-secrets).
+
+</TabItem>
+</TabGroup>
+
+</TabItem>
+</TabGroup>
+</TabItem>
+<TabItem name="Hybrid">
+
+### Hybrid
+
+To set up Hybrid deployment and deploy your code, you'll need to:
+
+1. Set up an agent
+2. Configure CI/CD for your project
+3. Add the code location in Dagster Cloud
+
+For most Hybrid deployments - with the exception of those using a local agent - you'll need to create a Docker image containing your Dagster code and then add a code location to Dagster Cloud that references the image.
 
 ---
 
-## Serverless Deployments
+**Step 1: Create a Hybrid agent**
 
-[Serverless Deployment](/dagster-cloud/deployment/serverless) is a fully-managed version of Dagster Cloud that allows you to run your Dagster jobs without spinning up any infrastructure. While Serverless is the easiest way to get started with Dagster, this deployment type works best with jobs that aren't computationally expensive.
+1. The modal that displays will contain a pre-generated [agent token](/dagster-cloud/account/managing-user-agent-tokens) and info about the agents currently supported by Dagster Cloud.
+2. Follow the steps for setting up and deploying your agent:
 
-Check out the [Getting started with Serverless guide](/dagster-cloud/getting-started/getting-started-with-serverless-deployment) to get up and running.
+   - [Amazon Elastic Container Service (ECS)](/dagster-cloud/deployment/agents/amazon-ecs/creating-ecs-agent-new-vpc)
+   - [Docker](/dagster-cloud/deployment/agents/docker)
+   - [Kubernetes](/dagster-cloud/deployment/agents/kubernetes/configuring-running-kubernetes-agent)
+   - [Local](/dagster-cloud/deployment/agents/local)
+
+Once Dagster Cloud receives an agent heartbeat, click **Continue**.
 
 ---
 
-## Hybrid Deployments
+**Step 2: Configure CI/CD for your project**
+
+Next, you'll configure CI/CD for your Dagster project. How this is accomplished depends on your CI/CD provider:
+
+- **GitHub** - If using GitHub, you can use our GitHub Action workflow to set up CI/CD for your project.
+- **Another CI/CD provider** - If you're not using GitHub, you can configure CI/CD using the dagster-cloud CLI.
+
+<TabGroup>
+<TabItem name="GitHub Actions">
+
+For continunous integration using GitHub actions:
+
+1. Copy the workflow files (`.github/workflows`) from the [Hybrid quickstart repository](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/tree/main/.github/workflows) and add them to your repository.
+
+2. In both workflow files, uncomment the step that is relevant to your Docker container registry. For example, if using DockerHub, you'd uncomment the `DockerHub` step in these files.
+
+3. In the repository, set the necessary GitHub action secrets:
+
+   - `DAGSTER_CLOUD_API_TOKEN` - A Dagster Cloud agent token. Refer to the [agent tokens documentation](/dagster-cloud/account/managing-user-agent-tokens#managing-agent-tokens) for more info.
```

Comment:
Is the recommendation for branch deps for Hybrid to still create a second, dedicated agent? https://docs.dagster.io/dagster-cloud/developing-testing/branch-deployments/using-branch-deployments-with-github#step-2-create-and-configure-an-agent

Reply:
I think most people do it all in a single agent now. It's something you can configure when you set up the agent (you have to explicitly tell it that it will be handling branch deployments). The easiest is probably to set that config from a single agent, but if you want your branch deployments to stay isolated from prod you could also run them in their own agent in a separate cluster.